internal/osvdev/models.go

```diff
@@ -28,12 +29,14 @@ type MinimalVulnerability struct {
 
 // Response represents a full response from OSV.
 type Response struct {
-	Vulns []models.Vulnerability `json:"vulns"`
+	Vulns         []models.Vulnerability `json:"vulns"`
+	NextPageToken string                 `json:"next_page_token"`
```

Comment:
do we decide to use the snake case? or camel case?

Reply:
this matches our API's response, which is in snake_case (since it's in python).